doc/template.html

```diff
@@ -4,8 +4,9 @@
   <meta charset="utf-8">
   <title>__SECTION__ Node.js __VERSION__ Manual &amp; Documentation</title>
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic">
+  <link href='https://fonts.googleapis.com/css?family=Fira+Mono:400,700' rel='stylesheet' type='text/css'>
```

Comment:
Is weight 700 used by the style at all? Also, could you put the `rel` attribute first and remove the `type`?